src/components/ResultsListItem.vue

```diff
@@ -11,7 +11,11 @@
 <script>
 export default {
   props: ['result'],
-  inject: ['savePark'],
+  methods: {
+    savePark() {
+      this.$store.commit('savePark', this.result.name)
```

Comment:
Something to note:

Remember how React uses `this.props` for class-based components and `props` for functional components?
Vue does something similar.  `$store` is how to access the store in a functional component, while `this.$store` is the syntax for components that have their own data.

Reply:
Thanks for these comments explaining how this works.